### PR TITLE
Validated exam authorizations were accepted by Pearson 

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
fixes https://github.com/mitodl/micromasters/issues/2080

#### What's this PR do?
- A task that open ftp and process Exam Authorization Confirmation files (EAC) files.
- In case of success or failure of ``Exam Authorization`` it updates status of ExamAuthorization object.

#### Note:
- Flag need to set is ```EXAMS_SFTP_TEMP_DIR=(to a dir in you local machine)```

@pdpinch

### CODE REVIEW

File: exams/pearson/download.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    results = EACReader().read(local_path)
+    messages = []
+    response = False
+    for result in results:
+        try:
+            exam_authorization = ExamAuthorization.objects.get(id=int(result['exam_authorization_id']))
+            if result['status'] == EAC_SUCCESS_STATUS:
+                exam_authorization.status = ExamAuthorization.STATUS_SUCCESS
+            else:
+                exam_authorization.status = ExamAuthorization.STATUS_FAILED
```
Comment (amir-qayyum-khan) [inline exams/pearson/download.py]: @rhysyngsun is this approach right?

**Comment by pdpinch** [comment]
@rhysyngsun what should happen with this PR?

**Comment by rhysyngsun** [comment]
@amir-qayyum-khan @pdpinch this code is now included in a new PR #2447 so this one can be closed.

File: exams/tasks.py [hunk 2]
```diff
@@ -135,3 +130,16 @@ def export_exam_authorizations(self):
                 id__in=valid_auth_ids).update(status=ExamAuthorization.STATUS_IN_PROGRESS)
         except:  # pylint: disable=bare-except
             log.exception('Unexpected exception updating ExamProfile.status')
+
+
+@async.task
+def batch_process_eac_files():
+    """
```
Comment (rhysyngsun) [inline exams/tasks.py]: I've addressed this in the other PR.

**Comment by amir-qayyum-khan** [comment]
merged in #2447

**Pull request closed without merging**